eventgrid/spring-cloud-azure-starter-eventgrid/spring-cloud-azure-sample-eventgrid/README.md

```diff
@@ -0,0 +1,86 @@
+---
+page_type: sample
+languages:
+- java
+products:
+- event-grid
+name: Use EventGrid to send EventGridEvent and use Service Bus Queue as Event Handlers to receive in a Spring Boot application.
+description: This sample demonstrates how to use EventGrid to send EventGridEvent and use Service Bus Queue as Event Handlers to receive in a Spring Boot application.
+---
+
+# Use EventGrid to send EventGridEvent and use Service Bus Queue as Event Handlers to receive in a Spring Boot application
+
+This sample demonstrates how to use EventGrid to send EventGridEvent to a topic and use Service Bus Queue as Event Handlers to receive.
+
+## What You Will Build
+You will build an application with EventGrid to send EventGridEvent to a topic and use Service Bus Queue as Event Handlers to receive.
+
+## What You Need
+
+- [An Azure subscription](https://azure.microsoft.com/free/)
+- [JDK8](https://www.oracle.com/java/technologies/downloads/) or later
+- Maven
```

Comment:
Can we use maven wrapper instead? Or does this repo all use Maven directly?

Reply:
This repo all use Maven, shall we replace them all?

Reply:
Not now.